app/src/main/java/com/gerardojim/nuggetscalculator/ui/main/view/CalculateFragment.kt

```diff
@@ -1,39 +1,45 @@
 package com.gerardojim.nuggetscalculator.ui.main.view
 
-import androidx.lifecycle.ViewModelProvider
 import android.os.Bundle
-import androidx.fragment.app.Fragment
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.ArrayAdapter
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.lifecycleScope
-import com.gerardojim.nuggetscalculator.databinding.MainFragmentBinding
+import com.gerardojim.nuggetscalculator.R
+import com.gerardojim.nuggetscalculator.databinding.CalculateFragmentBinding
 import com.gerardojim.nuggetscalculator.ui.main.intent.MainIntent
-import com.gerardojim.nuggetscalculator.ui.main.viewmodel.MainViewModel
+import com.gerardojim.nuggetscalculator.ui.main.viewUtil.checkedChanges
+import com.gerardojim.nuggetscalculator.ui.main.viewUtil.selectionChanges
+import com.gerardojim.nuggetscalculator.ui.main.viewUtil.textChanges
+import com.gerardojim.nuggetscalculator.ui.main.viewmodel.CalculateViewModel
 import com.gerardojim.nuggetscalculator.ui.main.viewmodel.MainViewModelFactory
 import com.gerardojim.nuggetscalculator.ui.main.viewstate.MainState
-import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.*
 import kotlinx.coroutines.launch
 
 class CalculateFragment : Fragment() {
-
-    private lateinit var viewModel: MainViewModel
-    private lateinit var binding: MainFragmentBinding
+    private lateinit var viewModel: CalculateViewModel
+    private lateinit var binding: CalculateFragmentBinding
```

Comment:
Isn't this supposed to take you to an XML when you cmd + click on it?

Reply:
@kelveenfabian is it? You might be thinking of databinding